apps/dot_ext/oauth2_backends.py

```diff
@@ -17,8 +18,9 @@ def create_token_response(self, request):
         """
         # Get session values previously stored in AuthFlowUuid from AuthorizationView.form_valid() from code.
         body = dict(self.extract_body(request))
-        code = body.get('code', None)
-        update_session_auth_flow_trace_from_code(request, code)
+        clear_session_auth_flow_trace(request)
+        update_session_auth_flow_trace_from_code(request, body.get('code', None))
+        set_session_auth_flow_trace_value(request, 'auth_grant_type', body.get('grant_type', None))
```

Comment:
Just checking but shouldn't `set_session_auth_flow_trace_value` move above `update_session_auth_flow_trace_from_code` so the `auth_grant_type` value can get propagated to the model object?

Reply:
> Just checking but shouldn't `set_session_auth_flow_trace_value` move above `update_session_auth_flow_trace_from_code` so the `auth_grant_type` value can get propagated to the model object?

`auth_grant_type` didn't need to be included in the AuthFlowUuid model for tracking. It is at the very end of the logging and that shows up in the final token related logging.